app/views/_includes/feedback-banner.nunjucks

```diff
@@ -0,0 +1,46 @@
+<div class="notification-banner notification-banner__hiviz">
+  <p class="notification-banner--inner">
+    This is a new page. <a href="#">Tell us what you think</a> or go
+    <a href="#">back to NHS Choices.</a>
+  </p>
+</div>
+
+<div class="feedback-banner">
```

Comment:
Can we decouple the component name from the content. `feedback` feels very specific.